src/components/DropdownContent/index.tsx

```diff
@@ -12,13 +14,27 @@ function createEnum<T extends { [P in keyof T]: P }>(o: T) {
   return o
 }
```

Comment:
I think, we can remove the `TKeys` and the `createEnum` entities because of useless themselves.